tests/checks/mock/test_fail2ban.py

```diff
@@ -0,0 +1,167 @@
+# stdlib
+import unittest
+import subprocess
+
+# third party
+import mock
+
+# project
+from tests.checks.common import AgentCheckTest, get_check
+
+
+class Fail2BanTestCase(unittest.TestCase):
+    def setUp(self):
+        self.config = """
+init_config:
+
+instances:
+   - sudo: True
+"""
+
+    def _get_check(self):
+        check, _ = get_check("fail2ban", self.config)
```

Comment:
We'd like `5.5.0` tests to favor [load_check](https://github.com/DataDog/dd-agent/blob/745d0ccd4ae8d9bcf4aa87540e552f8ae744c275/tests/checks/common.py#L44) over `get_check`. This will additionally require you to pass in test config as dictionaries instead of a yaml string.